### PR TITLE
Intro fixes

### LINKED ISSUE #49: Toreless - draft-5 section 1: fixes to Introduction
````
95	1.  Introduction

97	   Bootstrapping Remote Secure Key Infrastructures [BRSKI] specifies
98	   automated bootstrapping of an Autonomic Control Plane.  BRSKI

Suggest to replace after BRSKI with:

  ... BRSKI specifies automated and secure provisioning  of nodes (which are called pledges)
  with cryptographic keying material (trust anchors and certificates) to enable authenticated and
  confidential communication with other similarily enrolled nodes. This is also called enrolment.

No need to mention Autonomic Control Plane here IMHO (just sounds like limiting scope
of BRSKI Cloud).

If you really want folks to understand what is happening here, i suggest to
include explanations such as the following.

In BRSKI, the pledge performs enrolment by communicating with a BRSKI Registrar
belonging to the domain that provides the cryptopgraphic keying material and
usually is or acts upon the owner of the pledge. The pledge does not know 
who its owner is. Insted, in BRSKI it is assumed that the network to which the
pledge connects belongs to the owner of the pledge and therefore network-supported
discovery mechanisms can resolve generic, non-owner specific names to the owners Registrar.
To support enrolment of pledges without such an owner based access network, the mechanisms
of BRSKI Cloud are required which assume that Pledge and Registrar simply connect to the
Internet. The Internet ("Cloud") connected Registrar will then determine ownership of the Pledge
and redirect the Plege to its owners Registar.

````

### PR DESCRIPTION
**Pull request by mcr**
close #49

### CODE REVIEW

File: anima-brski-cloud.md [hunk 1]
```diff
@@ -50,8 +49,21 @@ To this, this document defines how to contact a well-known Cloud Registrar, and
...
 # Introduction
 
-Bootstrapping Remote Secure Key Infrastructures {{BRSKI}} and {{RFC8994}} specifies automated network onboarding of devices,  referred to as pledges, within an Autonomic Control Plane or other managed network infrastructure.
-BRSKI Section 2.7 describes how a pledge "MAY contact a well-known URI of a Cloud Registrar if a local Registrar cannot be discovered or if the pledge's target use cases do not include a local Registrar".
+Bootstrapping Remote Secure Key Infrastructures {{BRSKI}} BRSKI specifies automated and secure provisioning  of nodes (which are called pledges) with cryptographic keying material (trust  anchors and certificates) to enable authenticated and confidential communication with other similarily enrolled nodes.
+This is also called enrolment.
+
+In BRSKI, the pledge performs enrolment by communicating with a BRSKI Registrar
+belonging to the owner of the pledge.
+The pledge does not know who its owner will be when manufacturer.
```
Comment (upros) [inline anima-brski-cloud.md:57]: Typo: "when manufacturer." should be " when manufactured."

File: anima-brski-cloud.md [hunk 2]
```diff
@@ -50,8 +49,21 @@ To this, this document defines how to contact a well-known Cloud Registrar, and
...
+Instead, in BRSKI it is assumed that the network to which the pledge connects belongs to the owner of the pledge and therefore network-supported discovery mechanisms can resolve generic, non-owner  specific names to the owners Registrar.
+
+To support enrolment of pledges without such an owner based access network, the mechanisms
+of BRSKI Cloud are required which assume that Pledge and Registrar simply connect to the
+Internet.
+The Internet ("Cloud") connected Registrar will then determine ownership of the Pledge
+and redirect the Plege to its owners Registar.
+
+THis work is in support of {{BRSKI, Section 2.7}}, which describes how a pledge "MAY contact a well-known URI of a Cloud Registrar if a local Registrar cannot be discovered or if the pledge's target use cases do not include a local Registrar".
```
Comment (upros) [inline anima-brski-cloud.md:66]: Typo: "THis" should be "this"

**Review by upros — CHANGES_REQUESTED** [review]
Identified two minor nits.

File: anima-brski-cloud.md (hunk 1 above)
Comment (mcr) [inline anima-brski-cloud.md:57]:
```suggestion
The pledge does not know who its owner will be when manufactured.
```

File: anima-brski-cloud.md (hunk 2 above)
Comment (mcr) [inline anima-brski-cloud.md:66]:
```suggestion
This work is in support of {{BRSKI, Section 2.7}}, which describes how a pledge "MAY contact a well-known URI of a Cloud Registrar if a local Registrar cannot be discovered or if the pledge's target use cases do not include a local Registrar".
```

**Review by upros — APPROVED** [review]

**Pull request merged**